### ISSUE TITLE
Using Dropzone.js & Sails beta w/ skipper & skipper s3

### ISSUE THREAD

**Issue by ptariche** [balderdashy/skipper#63]
I receive the following errors while running debug:

 skipper Set up "maxTimeToWaitForFirstFile" timer for 10000ms +0ms
Acquiring new Upstream for field `file[0]`
  skipper passed control to app because first file was received +1ms
  skipper waiting for any text params +1ms
Upstream: Pumping incoming file through field `file[0]`
  skipper Something is trying to read from Upstream `file[0]`... +6ms
  skipper Passing control to app... +2ms
  skipper Set up "maxTimeToWaitForFirstFile" timer for 10000ms +3ms
Acquiring new Upstream for field `file`
  skipper .upload() called on upstream +147ms
  skipper Something is trying to read from Upstream `file`... +2ms
  skipper maxTimeToWaitForFirstFile timer fired- as of now there are 1 file uploads pending (so it's fine) +10s
  skipper maxTimeToWaitForFirstFile timer fired- as of now there are 0 file uploads pending  +16ms
Fatal incoming error in Upstream `file` ::   (source or user may have cancelled the request)
Error: ETIMEOUT: An Upstream (`file`) timed out waiting for file(s). No files were sent after waiting 10000ms.
-----------
ETIMEOUT
'ETIMEOUT: An Upstream (`file`) timed out waiting for file(s). No files were sent after waiting 10000ms.' - 'Error'
-----------
  skipper Upstream: No more files will be sent through field `file`- clearing timeouts... +1ms
  skipper upstream emitted error, forcing us to trigger the callback for .upload() with err: Error: ETIMEOUT: An Upstream (`file`) timed out waiting for file(s). No files were sent after waiting 10000ms. +1ms
error: Sending 500 ("Server Error") response:
 Error: ETIMEOUT: An Upstream (`file`) timed out waiting for file(s). No files were sent after waiting 10000ms.
    at null.<anonymous> (/Users/ptariche/Documents/projects/yarnreel-api/node_modules/skipper/standalone/Upstream/Upstream.js:62:15)
    at Timer.listOnTimeout [as ontimeout] (timers.js:112:15) { [Error: ETIMEOUT: An Upstream (`file`) timed out waiting for file(s). No files were sent after waiting 10000ms.]
  code: 'ETIMEOUT',
  message: 'ETIMEOUT: An Upstream (`file`) timed out waiting for file(s). No files were sent after waiting 10000ms.' }
A receiver is finished writing files from Upstream `file`.
(this doesn't necessarily mean any files were actually written...)
  skipper maxTimeToBuffer timer fired- upstream is NOT CONNECTED TO A RECEIVER!! As of now there are 1 file uploads +484ms
Fatal incoming error in Upstream `file[0]` ::   (source or user may have cancelled the request)
Error: EMAXBUFFER: An Upstream (`file[0]`) timed out before it was plugged into a receiver. It was still unused after waiting 10500ms. You can configure this timeout by changing the `maxTimeToBuffer` option.
-----------
EMAXBUFFER
'EMAXBUFFER: An Upstream (`file[0]`) timed out before it was plugged into a receiver. It was still unused after waiting 10500ms. You can configure this timeout by changing the `maxTimeToBuffer` option.' - 'Error'
-----------
  skipper Unconnected upstream: Encountered read error on incoming file `test.jpg` :: { [Error: EMAXBUFFER: An Upstream (`file[0]`) timed out before it was plugged into a receiver. It was still unused after waiting 10500ms. You can configure this timeout by changing the `maxTimeToBuffer` option.]
  code: 'EMAXBUFFER',
  message: 'EMAXBUFFER: An Upstream (`file[0]`) timed out before it was plugged into a receiver. It was still unused after waiting 10500ms. You can configure this timeout by changing the `maxTimeToBuffer` option.' } +1ms
  skipper Upstream: No more files will be sent through field `file[0]`- clearing timeouts... +0ms

**Comment by vinhtq** [comment]
I have the same problem. I think there is bug may be in both side: dropzonejs or skipper

You still can upload the file if you set
```
uploadMultiple: false
```
in dropzonejs. Everything works but you cannot upload multiple file

The second is may be skipper cannot receive multiple files upload as an array!
Because when you set
```
uploadMultiple: true
````
in dropzonejs, it will auto add [] to the params. 

Example: 
```
<input type="file" name="files" />
```
You server will receive params like this: files[]

**Comment by metzuda** [comment]
@ptariche Did you get this figured out?

**Comment by sailsbot** [comment]
Thanks for posting, @ptariche.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
